source/classes.tex

```diff
@@ -1212,6 +1212,13 @@
 and, if the class is not abstract\iref{class.abstract}, its virtual base
 classes are called its \term{potentially constructed subobjects}.
 
+\pnum
+A defaulted special member function is
+\indextext{function!member!special!defaulted!constexpr-compatible}%
```

Comment:
This gives a five-level deep index entry. We don't want that.
This "function index" area needs a cleanup, but for the time being, please just use "member function!constexpr-compatible".